packages/website/src/components/linter/utils.ts

```diff
@@ -17,3 +19,48 @@ export function createURI(marker: Monaco.editor.IMarkerData): string {
     (typeof marker.code === 'string' ? marker.code : marker.code?.value) ?? '',
   ].join('|')}]`;
 }
+
+export function parseMarkers(
+  markers: Monaco.editor.IMarker[],
+  fixes: Map<string, LintCodeAction[]>,
+  editor: Monaco.editor.IStandaloneCodeEditor,
+): ErrorItem[] {
+  return markers.map(marker => {
+    const isTypescript = marker.owner === 'typescript';
+    const code =
+      typeof marker.code === 'string' ? marker.code : marker.code?.value ?? '';
+    const uri = createURI(marker);
+
+    const fixers =
+      fixes.get(uri)?.map(item => {
+        return {
+          message: item.message,
+          fix(): void {
+            const model = editor.getModel()!;
+            const start = model.getPositionAt(item.fix.range[0]);
+            const end = model.getPositionAt(item.fix.range[1]);
+            editor.executeEdits(model.getValue(), [
+              {
+                range: {
+                  startLineNumber: start.lineNumber,
+                  startColumn: start.column,
+                  endLineNumber: end.lineNumber,
+                  endColumn: end.column,
+                },
+                text: item.fix.text,
+              },
+            ]);
+          },
+        };
+      }) ?? [];
+
+    return {
+      group: isTypescript ? marker.owner : code,
```

Comment:
Nit: it's a little odd seeing lower-case _typescript_ instead of _TypeScript_.

![Screenshot of two sections in the error viewer, "@typescript-eslint/prefer-as-const" and "typescript" (both lowercase)](https://user-images.githubusercontent.com/3335181/170183822-c27e8349-8f14-4289-b6ef-54c3931a1176.png)

```suggestion
      group: isTypescript ? 'TypeScript' : code,
```